arm-datalake-analytics/account/2016-11-01/swagger/account.json

```diff
@@ -499,7 +499,6 @@
           {
             "name": "parameters",
             "in": "body",
-            "required": true,
```

Comment:
To add some context, the reason I would like to remove the "required" tag is because the body is an object with all optional parameters. Nothing in the body itself is required, so it is weird for a user of the SDK to have to do something like this:
DataLakeStoreAccounts.Add(myResourceGroup, myAdlaAccount, accountNameToAdd, new AccountAddParameters());